apps-rendering/src/capi.ts

```diff
@@ -48,9 +48,6 @@ const isPhotoEssay = (content: Content): boolean =>
 const isFeature = (content: Content): boolean =>
 	content.tags.some((tag) => tag.id === 'tone/features');
 
-const isReview = (content: Content): boolean =>
-	[0, 1, 2, 3, 4, 5].includes(content.fields?.starRating ?? -1);
-
```

Comment:
This function was only being used to perform a check in `relatedContent.ts`.
A separate function checking for tags  `tone/review`, `tone/livereview`, `tone/albumreview` already exists in `item.ts` so this is now redundant).